src/commands/indexer/index.ts

```diff
@@ -934,7 +956,7 @@ export default class Indexer extends HealthCheck {
           } catch (error: any) {
             this.networkMonitor.structuredLogError(
               job.network,
-              this.errorColor(`Decoding CrossChainMessageSentEvent error: `, error),
+              this.errorColor(`Handling CrossChainMessageSentEvent error: `, error),
```

Comment:
This doesn't actually do anything to handle the error besides logging it right?

Reply:
yep